CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [3.0.0] - 2019-02-28
+
+### Added
+- Added `automatic_software_updates` resource to enable or disable the automatic checking, downloading, and installing of software updates.
+- Added `azure-pipelines.yml` to allow for managing builds as code.
+- Added some resource unit tests for `spotlight` to complement the existing `metadata_util` tests.
+
+### Changed
+- Changed the `ard` resource to `remote_management` and updates applicable tests and documentation. The new `remote_management` resource greatly simplifies syntax and reduces the needed macOS domain knowledge around `kickstart` options. However, it has less functionality than `ard` and is a significant breaking change.
```

Comment:
Did we forgo moving `keep_awake` to macos_test?

Reply:
Yea.